.travis.yml

```diff
@@ -18,14 +18,16 @@ cache:
     - vendor
 
 stages:
-  - dep-cache
-  - test
+  - name: dep-cache
+    if: tag != dev
```

Comment:
we dont need to cache for `dev`?

Reply:
We are introducing `dev` tag in the `release-dev` stage. When the new tag appears Travis starts a new build for this tag. This condition just prevents this run the second time.

Reply:
Sounds very complicated, how to make readable about this side effect?